codingStandards/CodingStandard-Html.adoc

```diff
@@ -19,11 +19,10 @@ Do *NOT* include xml namespace in the document.
 ----
 
 * *Character Encoding*
-All markup should be delivered as UTF-8, as its the most friendly for internationalization.
+All markup should be delivered via the following HTML element, and also as UTF-8, as its the most friendly for internationalization.
```

Comment:
`as its` -> `as it is` or `as it's`?

Reply:
I think `and also` may cause confusion. The use of UTF-8 should remain the focus of that sentence.
This will do: `All markup should be delivered via the following HTML element as UTF-8, as it is the most friendly for internationalization.`

Reply:
Ah I was having trouble figuring how to make sure the point was conveyed clearly. Thanks!